webapp/src/app/shared/widgets/string-widget/string-widget.component.html

```diff
@@ -1,13 +1,17 @@
 <div fxLayout="row" class="property-container" [matTooltip]="attribute?.description">
 	<label [for]="key" class="property-name">{{key}}:</label>
 	<div class="mat-select-container" *ngIf="attribute?.options; else defaultInput">
-		<mat-select [id]="key"
-		            [ngModel]="settings[key]"
-		            (selectionChange)="setSetting(key, $event.value)">
-			<mat-option *ngFor="let option of keys" [value]="option">
-				{{ option }}
+		<input class="default-input"
+			[id]="key"
+			[matAutocomplete]="auto"
+			[ngModel]="settings[key]"
+			(ngModelChange)="setSetting(key, $event); updateSuggestedOptions()"
+		>
+		<mat-autocomplete #auto="matAutocomplete" class="dark-scrollbar">
```

Comment:
This option would be nice
```suggestion
		<mat-autocomplete #auto="matAutocomplete" class="dark-scrollbar" [autoActiveFirstOption]="true">
```